estuary/api/v1.py

```diff
@@ -240,14 +242,21 @@ def get_siblings(resource, uid):
     :raises NotFound: if the item is not found
     :raises ValidationError: if an invalid resource was requested
     """
+    story_type_mapper = {'container': 'ContainerStoryManager', 'module': 'ModuleStoryManager'}
+    story_type = request.args.get('story_type', 'container').lower()
     # This is the node that is part of a story which has the relationship to the desired
     # sibling nodes
     story_node = get_neo4j_node(resource, uid)
     if not story_node:
         raise NotFound('This item does not exist')
 
-    # Defaulting to Container Story until we modify this endpoint to suit the module changes
-    story_manager = ContainerStoryManager()
+    story_manager_class = story_type_mapper.get(story_type)
```

Comment:
Can you move this code to before the API queries Neo4j (up 5 or so lines)?

Reply:
To clarify, this is because we shouldn't burden Neo4j with an unnecessary query if the request is invalid.